rust/eos-rs/src/api/fixed.rs

```diff
@@ -0,0 +1,95 @@
+//! Dealing with [fixed-point numbers](https://en.wikipedia.org/wiki/Fixed-point_arithmetic)
+//! used in the game.
+//!
+//! Note that this module currently only deals with binary fixed-point representations.
+//! The game also sometimes uses decimal representations of fixed-point numbers
+//! (eg. 0x64 -> 100 -> '01.00').
+//!
+//! This pulls in parts of the [`fixed`](https://docs.rs/fixed/latest/fixed/index.html) crate,
+//! which describes these numbers as follows:
+//!
+//! > An <i>n</i>-bit fixed-point number has <i>f</i>&nbsp;=&nbsp;`Frac` fractional
```

Comment:
Ew, HTML is hard to read within source code.

Reply:
I agree, but that's how they did it :sob: 
I guess I can just link there instead.

Reply:
Most IDE format these nicely if you hover over the function, so I don't mind it

Reply:
Yeah I guess that's fine 🤷. It's not super important at any rate.